example/src/main.rs

```diff
@@ -1,40 +1,24 @@
-#[macro_use]
-extern crate cdrs;
-#[macro_use]
-extern crate cdrs_tokio_helpers_derive;
-extern crate time;
-
-use time::Timespec;
 use std::collections::HashMap;
-use cdrs_tokio::types::AsRustType;
-use cdrs_tokio::types::value::{Bytes, Value};
-use cdrs_tokio::frame::{IntoBytes, TryFromRow, TryFromUDT};
-use cdrs_tokio::types::rows::Row;
-use cdrs_tokio::types::udt::UDT;
-use cdrs_tokio::types::list::List;
-use cdrs_tokio::types::map::Map;
-use cdrs_tokio::types::from_cdrs::FromCDRSByName;
+use cdrs_tokio::query_values;
 
-// #[derive(Debug, IntoCDRSValue, TryFromRow)]
-#[derive(Clone, Debug, IntoCDRSValue, TryFromRow)]
+#[derive(Clone, Debug, cdrs_tokio_helpers_derive::IntoCDRSValue, cdrs_tokio_helpers_derive::TryFromRow)]
```

Comment:
I think the idiomatic way would be to `use cdrs_tokio_helpers_derive::*` instead of full paths.